README.md

```diff
@@ -11,6 +11,10 @@
 <a href="https://oras.land/"><img src="https://oras.land/img/oras.svg" alt="banner" width="100px"></a>
 </p>
 
+`oras-credentials-go` is a credential management library designed for [`oras-go`](https://github.com/oras-project/oras-go). It supports reading, saving, and removing credentials from Docker configuration files and external credential stores that follow the [Docker credential helper protocol](https://docs.docker.com/engine/reference/commandline/login/#credential-helper-protocol). It does not handle credential encryption and decryption.
+
+Once stable, this library will be merged into `oras-go`. (See [discussion](https://github.com/oras-project/oras-credentials-go/discussions/80))
```

Comment:
```suggestion
Once it has a fairly stable version, this library will be merged into `oras-go`. (See [discussion](https://github.com/oras-project/oras-credentials-go/discussions/80))
```

Reply:
In general, `Stable` means v1.0.0 but oras-credential-go will only cut a pre-release (v1.0.0-rc.1)

Reply:
How about 
> Once it reaches a fairly stable version (e.g. `v1.0.0-rc.1`), this library will be merged into `oras-go`?